sonar-html-plugin/src/main/java/org/sonar/plugins/html/api/accessibility/AccessibilityUtils.java

```diff
@@ -44,4 +46,12 @@ public static boolean isDisabledElement(TagNode element) {
     var ariaDisabledAttr = element.getAttribute("aria-disabled");
     return "true".equalsIgnoreCase(ariaDisabledAttr);
   }
+
+  public static boolean isFocusableElement(TagNode element) {
+    String tabindex = element.getAttribute("tabindex");
+    if (isInteractiveElement(element)) {
+      return tabindex == null || Double.parseDouble(tabindex) >= 0;
+    }
+    return tabindex != null && Double.parseDouble(tabindex) >= 0;
+  }
```

Comment:
There is a false positive for <input type="hidden"/>, it should not be focusable.
https://sonarcloud.io/project/issues?impactSoftwareQualities=RELIABILITY&resolved=false&types=CODE_SMELL&id=jhipster-sample-application&open=AY3niTmAHmRkDeY1zyU2

Reply:
that's been [fixed](https://github.com/SonarSource/sonar-html/pull/291) and will appear in the next release.

Reply:
The next release for VSCode v4.4.2 came out a few days ago but still appears to be an issue